### ISSUE TITLE
Error using plugin with blockContent

### ISSUE THREAD

**Issue by LeonardoRick** [sanity-io/sanity-plugin-internationalized-array#15]
I can't use `internationalizedArrayBlockContent` without getting the following error everytime I type something.

![image](https://github.com/sanity-io/sanity-plugin-internationalized-array/assets/17517057/c51996f8-412b-4c30-928c-30ee07175ec8)

It works properly with the text field, so thanks for that! But it's a very annoying bug, even not blocking me to continue producing content.

This is my config: 

```javascript
  internationalizedArray({
    languages,
    fieldTypes: ['string', 'blockContent'], // internationalizedArray<type>
  }),
```

**Comment by SimeonGriggs** [comment]
Can you share your schema for `blockContent`?

**Comment by LeonardoRick** [comment]
It's the one from the sanity blog boilerplate: https://github.com/cre8tive-studio/cms-leonardorick/blob/main/schemas/blockContent.ts

**Comment by andresclua** [comment]
hey guys, I'm having the same problem.
If I define this

```
defineField({
      name: 'field_test2',
      type: 'internationalizedArrayblockContent',
    }),
```
I have "Unknown type: internationalizedArrayblockContent."

**Comment by SimeonGriggs** [comment]
I'll look into this error, however, as stated in the [Localization Docs](https://www.sanity.io/docs/localization#0c676ad65e70), it is not recommended to use field-level translation for Portable Text fields

**Comment by LeonardoRick** [comment]
@SimeonGriggs thats fine, I'm using basic text for now but would be nice to have it since it's something present on the demo sanity project.

If you find out that it's really not supported maybe add some note on the plug-in instructions just for people to be aware.

**Comment by josephine-jc** [comment]
We would also like to use this plugin with block content. 

We get that this is not recommend for now, but we do have a mix of common fields and language specific fields, so using the the document language mode would mean a lot of duplicate information for us. 

So right now there doesn't seem to be a good solution for our wishes for multi-language in Sanity.

**Comment by SimeonGriggs** [comment]
The current issue with block content appears to be with the Portable Text Editor input in newer versions of the Studio. Once resolved this plugin should begin to work again with Portable Text.

**Comment by matthewborgman** [comment]
> The current issue with block content appears to be with the Portable Text Editor input in newer versions of the Studio. Once resolved this plugin should begin to work again with Portable Text.

@SimeonGriggs is there a GH issue to track for fixing the PTE inout in Studio? https://github.com/sanity-io/sanity/issues/4576 hasn't even been acknowledged.

**Comment by SimeonGriggs** [comment]
Fixed in [1.10.0](https://github.com/sanity-io/sanity-plugin-internationalized-array/releases/tag/v1.10.0)
